src/main/java/AddCommand.java

```diff
@@ -0,0 +1,23 @@
+import java.io.IOException;
+
+public class AddCommand extends Command{
+
+    private String[] commands;
+
+    public AddCommand(String commandWord, String[] commands) {
+        super(commandWord);
+        this.commands = commands;
+    }
+
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Let's have some JavaDoc comments for your methods :) (applies to the remaining code as well)